src/brogue/Rogue.h

```diff
@@ -2608,6 +2610,19 @@ typedef struct buttonState {
     cellDisplayBuffer rbuf[COLS][ROWS]; // Reversion screen state.
 } buttonState;
 
+enum messageFlags {
+    REQUIRE_ACKNOWLEDGMENT        = Fl(0),
+    REFRESH_SIDEBAR               = Fl(1),
```

Comment:
Do you know the need for this flag? Is it bad if we always refresh when a message is drawn?

Reply:
It ties into the next commit, 58b1fc6 , where we replace a use of `message` with `temporaryMessage` in `getInputTextString`.  `temporaryMessage` declares itself meant for prompts and, indeed, using `message` in `getInputTextString` to draw the prompt was causing problems.  

`getInputTextString` is used to prompt for various filenames, including in screens where the main game interface is not displayed.  As such we would not want it to (always) redraw the sidebar.  Notice that 

`temporaryMessage` used to redraw the sidebar unconditionally, which may have been the main reason it was not used in `getInputTextString`.  The sidebar-redrawing behavior of `temporaryMessage` is important, I suspect, in its uses for prompting the player to choose a direction for throwing or zapping a staff.

I suspect the work of threading yet another positional boolean argument (a la `requireAcknowledgement`) may have dissuaded earlier development going back and cleaning up what otherwise seems to me to be a pretty clear use for `temporaryMessage`.  However with `messageFlags` that heavy lifting is somewhat easier.

So, `REFRESH_SIDEBAR` now tells `temporaryMessage` whether or not to redraw the sidebar, enabling it for use in prompts where we don't want it seen.  Since `mesasge` also can conditionally redraw the sidebar, it pays attention to the flag as well.